client/src/components/Catalog.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+
+class CourseTile extends Component {
+  render() {
+    return (
+      <li key={this.props.code}>
+        <ul className="course-tile">
+          <li>{this.props.code}</li>
+          <li>{this.props.name}</li>
+          <li>{this.props.midsem}</li>
+          <li>{this.props.compre}</li>
+        </ul>
```

Comment:
Are you following a standard or are you suggesting we follow this?

Reply:
Not really doing anything. This just made sense to me